packages/react/src/components/Pill/Pill.styles.ts

```diff
@@ -57,6 +57,17 @@ export const useStyles = css({
           backgroundColor: '$palette-red-600',
         },
       },
+      yellow: {
+        $$backgroundColor: '$colors$palette-yellow-100',
+
+        '.manifest-pill__text': {
+          color: '#BF7000',
```

Comment:
any of default palettte orange colors works? it's fine if not.

Reply:
<img width="773" alt="image" src="https://github.com/user-attachments/assets/f810bab4-8011-4e0e-b4b8-c65184d62162">

Reply:
@vitaliy-p44 yes it won't work . 
For reference- 
<img width="832" alt="Screenshot 2024-08-29 at 7 16 53 PM" src="https://github.com/user-attachments/assets/9a3db2ba-91e9-4817-8e1a-7abceef01ad3">